.github/workflows/java-gradle-release.yaml

```diff
@@ -55,6 +56,10 @@ on:
         description: "Bumped version of your project."
         value: ${{ jobs.release.outputs.release-version }}
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-java-gradle-release
+  cancel-in-progress: true
+
```

Comment:
We do not want workflows that create tags or releases to every be canceled while they are running